setup.py

```diff
@@ -111,8 +111,20 @@ def get_all_extras() -> Dict:
 with open(os.path.join(here, PACKAGE_NAME, "__version__.py"), "r") as f:
     exec(f.read(), about)
 
-with open("README.md", "r") as f:
-    readme = f.read()
+
+def parse_readme():
+    with open("README.md", "r") as f:
+        readme = f.read()
+
+    # replace relative links of images
+    raw_url_root = "https://raw.githubusercontent.com/fetchai/agents-aea/master/"
+    replacement = raw_url_root + r"\g<0>"
+    readme = re.sub(r"(?<=<img src=\")(/.*)(?=\")", replacement, readme, re.DOTALL)
```

Comment:
we take into consideration only the images whose `src` starts with `/`

... assuming there is exactly one space between `img` and `src` :D